client/users/user-profile.tsx

```diff
@@ -215,8 +221,9 @@ export function UserProfilePage({
   onTabChange,
   isAdmin,
 }: UserProfilePageProps) {
+  const { t } = useTranslation()
   // TODO(tec27): Build the title feature :)
-  const title = 'Novice'
+  const title = t('users.profile.novice', 'Novice')
```

Comment:
This should probably be under a `titles` category at least (e.g. `users.titles.novice` or something) given that we'll eventually have a bunch more of these